lib/cdo/i18n_string_url_tracker.rb

```diff
@@ -58,34 +58,55 @@ def initialize
   end
 
   # Records the given string_key and URL so we can analyze later what strings are present on what pages.
-  # @param string_key [String] The key used to review the translated string from our i18n system.
   # @param url [String] The url which required the translation of the given string_key.
   # @param source [String] Context about where the string lives e.g. 'ruby', 'maze', 'turtle', etc
-  def log(string_key, url, source)
+  # @param string_key [String] The key used to locate the desired string.
+  # @param scope [Array] Array of strings representing the hierarchy leading up to the string_key.
+  # @param separator [String] The separator string used by I18n to concatenate the string_key hierarchy
+  #        into a single normalized string.
+  def log(url, source, string_key, scope = nil, separator = nil)
```

Comment:
```suggestion
  def log(url, source, string_key, scope = [], separator = I18n.default_separator)
```
Looks like we are setting defaults down below, do this work instead?

Reply:
putting `separator = 1i8n.default_separator` wasn't working in the params for some reason, I figured maybe you can't make function calls inside the default params declaration? Setting scope to [] happened later so I just put it down with the separator, but yeah that one can probably be moved into the params